lib/ddr/index_fields.rb

```diff
@@ -74,6 +74,7 @@ def self.get(name)
     TECHMD_WELL_FORMED          = solr_name :techmd_well_formed, :symbol
     TITLE                       = solr_name :title, :stored_sortable
     WORKFLOW_STATE              = solr_name :workflow_state, :stored_sortable
+    YEAR_FACET                  = "year_facet_iim" # solr_name :year_facet, :facetable, type: :integer
```

Comment:
I would probably remove the trailing comment on this line since I assume 'year_facet_iim' is not what 'solr_name :year_facet, :faceable, type: :integer' produces